doc/source/ray-core/namespaces.rst

```diff
@@ -24,6 +24,14 @@ first connect to the cluster.
       System.setProperty("ray.job.namespace", "hello"); // set it before Ray.init()
       Ray.init();
 
+.. tabbed:: C++
+
+    .. code-block:: c++
+
+      ray::RayConfig config;
+      config.ray_namespace = "hello";
+      ray::Init(config);
+
```

Comment:
I believe you wrote the wrong style because I couldn't find your change on the temp doc of this PR:
https://docs.ray.io/en/latest/ray-core/namespaces.html
![image](https://user-images.githubusercontent.com/19473861/178986747-11a6ad5a-3be1-476a-bae3-d76d8db23884.png)

Reply:
I found a wrong link. Please miss my comment👆